client/src/modules/purchases/create/createUpdate.html

```diff
@@ -213,7 +213,24 @@ <h4 class="text-right" style="margin-right: 25px;">
               data-method="clear" translate>
               FORM.BUTTONS.CLEAR
             </button>
-            <p
+            <button
+              type="button"
+              class="btn btn-default"
+              ng-click="PurchaseCtrl.cancel()"
+              data-method="cancel"
+              ng-show="PurchaseCtrl.isUpdateState"
+              translate>
+              FORM.BUTTONS.CANCEL_EDIT
+            </button>
+            <button
+              type="button"
+              class="btn btn-default"
+              ng-click="PurchaseCtrl.cancel()"
+              data-method="cancel"
+              ng-show="!PurchaseCtrl.isUpdateState"
```

Comment:
These look identical, except for the `isUpdateState` call.  That means that you probably just need to change the text.  I propose you do something like:
```html
<button
  type="button"
  class="btn btn-default"
  ng-click="PurchaseCtrl.cancel()"
  data-method="cancel">
  <span ng-show="PurchaseCtrl.isUpdateState">FORM.BUTTONS.CANCEL_EDIT</span>
  <span ng-hide="PurchaseCtrl.isUpdateState">FORM.BUTTONS.CANCEL</span>
</button>
```

Do we really need both "clear" and "cancel"?  I don't think we need both.  Let's remove the other button and just put this button.

![image](https://user-images.githubusercontent.com/896472/121423172-e2ded180-c967-11eb-97ff-fab4017b573a.png)

Reply:
I Think the [Clear] button is only there when creating a PO, not when editing.   That seems right to me.  It maintains the [Clear] functionality that was already there (when creating) but removes it for editing, since that does not make as much sense to me.   If they want to remove everything while editing, they can do that-- but why not delete the PO?

I think the [Cancel] / [Cancel Edit] button are really useful since it is not always obvious what to do if you decide to abort creating or editing a PO.,

Regarding the duplicate labels--I'll fix that tomorrow.   I considered doing that but never got to it.

Reply:
This screenshot was taken during an edit - I think that the three buttons do show up when there is an edit.

Reply:
@jniles I've fixed the buttons to work like I wanted it to (see above) and incorporated the code cleanup you suggested.  I think it is ready to go now.   BTW, the reason I wanted the "cancel" button to read [Cancel Edit] when editing is to avoid confusion that that button would cancel the PO itself.